CalibCalorimetry/HcalAlgos/src/HcalLogicalMapGenerator.cc

```diff
@@ -1659,10 +1659,10 @@ void HcalLogicalMapGenerator::buildCALIBMap(const HcalTopology* topo,
       }
     }
   }
-  ic++;
+//ic++; // replace with ic=ic+1 below
```

Comment:
Please remove the comment since once the code is changed, there is no need for it